docker-compose.yml

```diff
@@ -65,6 +65,7 @@ services:
     ports:
       - "80:80"
       - "443:443"
+      - "2019:2019"
```

Comment:
same here, add a comment indicating local development setup uncomment this part and also updating the readme would be great

Reply:
don't need to commit this. readme should help  or add it as a comment